Makefile

```diff
@@ -27,7 +27,7 @@ lint:
 	${FLAKE8} phy --exclude=phy/ext/*,default_settings.py --ignore=E226,E265,F811
 
 test: lint
-	py.test --cov-report term-missing --cov phy --ignore experimental
+	py.test --cov-report term-missing --cov phy --ignore experimental -s
```

Comment:
the view tests might silently fail with error messages in the console: they are hidden without this option in py.test
need to improve those tests